site/ja/CONTRIBUTING.md

```diff
@@ -0,0 +1,78 @@
+# Contribution Guideline
+
+## Communities
+
+* Slack
+  * Slack の #doc_translation チャンネルで議論をしています
```

Comment:
typo? 
`#doc_translation` -> `#docs_translation`

Reply:
Thanks. I fixed it.